### PR TITLE
docs: add NextJS integration docs

### PR DESCRIPTION
**Pull request by barak007**
This should wait until the next Stylable release that support `filterAssets`

### CODE REVIEW

**Review by tomrav — APPROVED** [review]
Approved but waiting on Stylable release

File: docs/getting-started/nextjs-integration.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+id: nextjs-integration
+title: NextJS Integration
+---
+
+There are few steps you need to follow in order to integrate Stylable to your NextJS project.
```
Comment (elig-wix) [inline docs/getting-started/nextjs-integration.md:6]:
```suggestion
There are few steps you need to follow to integrate Stylable in your NextJS project.
```

File: docs/getting-started/nextjs-integration.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Create a `next.config.js`
+2. Use NextJS webpack 5 future
```
Comment (elig-wix) [inline docs/getting-started/nextjs-integration.md:9]:
```suggestion
2. Use the NextJS webpack 5 future
```

File: docs/getting-started/nextjs-integration.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+3. Exclude other css loading from the webpack configuration
```
Comment (elig-wix) [inline docs/getting-started/nextjs-integration.md:10]:
```suggestion
3. Exclude other CSS loaders from loading Stylable (*.st.css) files
```

File: docs/getting-started/nextjs-integration.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+id: nextjs-integration
+title: NextJS Integration
+---
+
+There are few steps you need to follow in order to integrate Stylable to your NextJS project.
+
+1. Create a `next.config.js`
```
Comment (elig-wix) [inline docs/getting-started/nextjs-integration.md:8]:
```suggestion
1. Create a `next.config.js` file
```

File: docs/getting-started/nextjs-integration.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+2. Use NextJS webpack 5 future
+3. Exclude other css loading from the webpack configuration
+4. Add the Stylable plugin to the webpack configuration
+
+### Example Configuration:
+
+```js
+const {
```
Comment (elig-wix) [inline docs/getting-started/nextjs-integration.md:16]:
```suggestion
// next.config.js
const {
```

**Pull request merged**